src/FlixterHeader.jsx

```diff
@@ -1,18 +1,22 @@
 import React, { useState } from "react";
 import Sidebar from "./Sidebar";
 import { doSignOut } from "./firebase/auth";
-
 import "./FlixterHeader.css";
+import { NavLink } from "react-router-dom";
+import logo from "./logo.png";
+
 function FlixterHeader({ likedMovies, watchedMovies }) {
   const [isSidebarOpen, setIsSidebarOpen] = useState(false);
   const toggleSidebar = () => {
     setIsSidebarOpen(!isSidebarOpen);
   };
   return (
     <header className="App-header">
-      <h1 className="title">🎥 Flixter 🎬</h1>
-      <div>
-        <button className="toggleSidebar" onClick={toggleSidebar}>
+      <NavLink className="title" to={"/home"}>
+        <img className="logo" src={logo} alt="" />
+      </NavLink>
+
+      {/* <button className="toggleSidebar" onClick={toggleSidebar}>
```

Comment:
Let's not keep commented code in the commits. This has to be removed and (if needed) recovered later from this same PR.